ToolsTest/Models/VladyslavClassArticleTests/EditorTestProperty_nUnit.cs

```diff
@@ -0,0 +1,51 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+using Tools.Models;
+
+namespace ToolsTest.Models.VladyslavClassArticleTests
+{
+    [TestFixture]
+    public class EditorTestProperty_nUnit
+    {
+        [Test]
+        public void Editor_HasTitle_NotNull ()
+        {
+            //Arrange
+            string title = "Title";
+            //Act
+            Article _editor = new Article(title);
+            //Assert
+            Assert.IsNotNull(_editor.HasTitle);
```

Comment:
The `HasTitle` property returns the boolean value that is the value type and this structure always returns not-null value. Better is just test that value is true instead.
```suggestion
            Assert.True(_editor.HasTitle);
```